src/Tasks/XmlPoke.cs

```diff
@@ -223,7 +223,14 @@ public override bool Execute()
 
             if (iter.Count > 0)
             {
+#if RUNTIME_TYPE_NETCORE
```

Comment:
Eww, this is unfortunate. And normally I'd say "define a specific feature flag instead!" but hopefully we can reunify this back in the .NETStandard 2.0 timeframe.